docs/specs/moniker.yml

```diff
@@ -750,15 +751,25 @@ inputs:
     monikerRange: '>= netcore-1.1'
     ---
     # [A](a.md)
-    ## [A](xref:a)
     ## [B](b.md)
+    ## [C](/c)
+    ## [D](http://test.com/test)
+    ### [A](xref:a)
+    ## [Toc ref](folder/TOC.yml)
```

Comment:
>## [Toc ref](folder/TOC.yml) [](start = 4, length = 28)

could you split this TOC inclusion case? there should be more to be checked in included TOC's monikers

Reply:
for example?

---
In reply to: [243783866](https://github.com/dotnet/docfx/pull/3797#discussion_r243783866) [](ancestors = 243783866)

Reply:
absolute url in included toc?

Reply:
sure, it can be extracted to another test case.

---
In reply to: [243784161](https://github.com/dotnet/docfx/pull/3797#discussion_r243784161) [](ancestors = 243784161)